libs/langchain-community/src/llms/replicate.ts

```diff
@@ -147,7 +168,8 @@ export class Replicate extends LLM implements ReplicateInput {
     Replicate: typeof import("replicate").default;
   }> {
     try {
-      const { default: Replicate } = await import("replicate");
+      // eslint-disable-next-line @typescript-eslint/no-var-requires,global-require
+      const Replicate = require("replicate");
```

Comment:
This will cause issue in some environments I think?

We actually don't need dynamic imports anymore, since we've added different entrypoints.

Reply:
`import` crashed for me in linux and osx with Jest. Is there a better way to do this?